packages/core/tocco-ui/src/EditableValue/editors/DatePicker.js

```diff
@@ -35,11 +36,12 @@ CustomTodayButton.propTypes = {
 }
 
 const TimeInput = ({value, onChange, onKeyDown}) => {
-  const handleOnChange = ev => {
+  const handleOnChange = val => {
     if (onChange) {
-      onChange(ev.target.value)
+      onChange(val || '00:00')
```

Comment:
this means, a time input can never be emptied, right?

depending on the use case, "00:00" vs null/empty could be different things.

Reply:
Yes, inside a DatePicker it cannot be emptied. otherwise the date gets invalid.  (see: https://toccoag.atlassian.net/browse/TOCDEV-5958)